modules/RTC/ortc/RTCSessionDescription.js

```diff
@@ -0,0 +1,86 @@
+import sdpTransform from 'sdp-transform';
+
+/**
+ * RTCSessionDescription implementation.
+ */
+export default class RTCSessionDescription {
+    /**
+     * RTCSessionDescription constructor.
+     * @param {object} [descriptionInitDict]
+     * @param {string} [descriptionInitDict.type] "offer" / "answer".
+     * @param {string|object} [descriptionInitDict.sdp] SDP string or
```

Comment:
I don't full understand what you mean. Passing a `sdp-transform` object will just be done from the Edge `RTCPeerConnection` shim. Code outside of it will never have to do that in any way.

BTW what should be named in a different way?